examples/01-maps/plot_sumMap.py

```diff
@@ -138,15 +138,15 @@ def run(plotIt=True):
     regMesh = TensorMesh([len(domains)])
 
     reg_m1 = regularization.Sparse(regMesh, mapping=wires.homo)
-    reg_m1.cell_weights = wires.homo * wr
+    reg_m1.set_weights(user_weights=wires.homo * wr)
```

Comment:
same comment as before
```suggestion
    reg_m1.set_weights(cell_weights=wires.homo * wr)
```